services/distribution/src/main/resources/main-config/v2/plausible-deniability-parameters.yaml

```diff
@@ -0,0 +1,5 @@
+#plausible-deniability-parameters:
```

Comment:
Why is this a separate file?

Reply:
We have this configuration in a separate file because the mapping between the yaml file and proto structure does not work as for the others objects. The problem is that the checkin-size-in-bytes is a list of Integer and the mapping does not work by default. I created a new file for this so we don't have to deserialise the whole presenceTracingParameters object.